shows/732 - Stop going to Spam.md

```diff
@@ -0,0 +1,43 @@
+---
+number: 732
+title: "Stop going to Spam: DMARC, SPF and DKIM Explained"
+date: 1708344000810
+url: https://traffic.libsyn.com/syntax/Syntax_-_732.mp3
+youtube_url: https://youtu.be/ERiQc8hoDvo
+---
+
+Wes and Scott explore the web standards and email authentication methods that keep your email from going to spam; DMARC, SPF and DKIM. If you send email via a custom domain name, you need to listen to this!
+
+### Show Notes
+
+* **[00:00](#t=00:00)** Welcome to Syntax!
+* **[01:06](#t=01:06)** Brought to you by Sentry.io.
```

Comment:
We should link Sentry up in the show notes as well - their SEO team likes this

Reply:
Ahhh great catch. Will do